src/sidebar/components/TopBar.tsx

```diff
@@ -101,9 +110,9 @@ function TopBar({
             <>
               {pendingUpdateCount > 0 && (
                 <IconButton
-                  icon="refresh"
+                  icon={RefreshIcon}
                   onClick={applyPendingUpdates}
-                  size="small"
+                  size="xs"
```

Comment:
This size change from small (sm) to xs is part of some [recent alignment changes](https://github.com/hypothesis/frontend-shared/pull/678) for sized components: all `xs` buttons have the same padding/spacing basis, all `sm`, etc. Previously, `IconButton` and `Button` were out of alignment, in that `IconButton`'s `sm` padding/spacing was smaller than `Button`'s. 

I don't think we've nailed sizing yet. Some of that imperfection is captured in [this issue](https://github.com/hypothesis/frontend-shared/issues/676), and I hope to iterate on this soonish.